### PR TITLE
Replace hard coding with existing constants

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
Replace hard coding with existing constants

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ZP-AlwaysWin** [comment]
> /lgtm

hi @cheftako  Thanks your code review。Can you help me with a /approve tag

**Comment by cheftako** [comment]
I would suggest getting deads2k or liggitt to look at and approve it. I would like them to check that they are ok with using the user agent string for the server cert name.

File: cmd/kube-controller-manager/app/options/options.go [hunk 1]
```diff
@@ -196,8 +197,8 @@ func NewKubeControllerManagerOptions() (*KubeControllerManagerOptions, error) {
 	}
 
 	s.GarbageCollectorController.GCIgnoredResources = gcIgnoredResources
-	s.Generic.LeaderElection.ResourceName = "kube-controller-manager"
-	s.Generic.LeaderElection.ResourceNamespace = "kube-system"
+	s.Generic.LeaderElection.ResourceName = KubeControllerManagerUserAgent
```
Comment (liggitt) [inline cmd/kube-controller-manager/app/options/options.go:200]: I would revert this one, it's not clear the user agent controls functional behavior like the lock resource name or key pair filename

Comment (stewart-yu) [reply cmd/kube-controller-manager/app/options/options.go:200]: i agree

**Pull request closed without merging**